coalib/misc/CachingUtilities.py

```diff
@@ -84,7 +84,7 @@ def pickle_load(log_printer, identifier, fallback=None):
                         Otherwise the ``default`` value is returned.
     """
```

Comment:
commit message: Checking -> Check
file_path - enclose in double backticks

same for "is"


in commit body: "Replacing" -> "Replaces"